cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -257,6 +257,33 @@
          {:indents {'thing.core/defn [[:inner 0]]}
           #?@(:cljs [:alias-map {"t" "thing.core"}])})
         "applies custom indentation to namespaced defn")
+    (testing "handles metadata on or comments before forms inside ns :require list"
+      (doseq [ignore-str [""
```

Comment:
Hope using `doseq` to test different combinations of stuff here is ok. Writing out 40 separate test cases seemed a bit noisy

Reply:
That's fine, but why did you use `doseq` instead of `are`?